webscraping/node/course-info/course-info-json-schema.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+
+    "definitions": {
+        "courseCodeString": {
+            "type": "string",
+            "pattern": "^[A-Z]{4} [0-9]{3,4}$"
+        },
+        "arrayOfCourseCodeOrLists": {
+            "type": "array",
+            "items": {
+                "type": "array",
+                "minItems": 1,
+                "items": { "$ref": "#/definitions/courseCodeString" }
+            }
+        },
+        "alphaNumPunctString": {
+            "type": "string",
+            "pattern": "^[\\wñüúáàíÎöóòëéÉèÈ ,'“”‘’\"--­‐‑‒–—/.:;?§()\\[\\]]+$"
+        },
+        "posNumRoundedToHundredths": {
+            "type": "string",
+            "pattern": "^[0-9](\\.[0-9]{1,2})?$"
```

Comment:
I would prefer to round to nearest tenth if that's possible, to save space (no credits include a third sig-fig, right?). Low priority doe.

Reply:
I mean there's stuff like 3.75 credits, I don't see why we would want it as 3.7

Reply:
Brutal

Reply:
yep.  hopefully if we ever bring this project to concordia's attention, we can perhaps convince them simply to *give us* the clean data that the `webscraping` tries so hard to make for us

Reply:
Also, once we're stuck doing floating-point math the extra decimal place really doesn't matter.